app/main.py

```diff
@@ -1,3 +1,55 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os
+
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+
+CONFIG_FILE = "config.json"
```

Comment:
Make sure you write the correct path to the JSON file. The current path is relative and may not work if the script is executed from a different directory.